code/modules/vehicles/vehicle_key.dm

```diff
@@ -55,7 +55,7 @@
 	if(user)
 		user.remove_atom_colour(ADMIN_COLOUR_PRIORITY)
 		user.visible_message("<span class='suicide'>[user] forgot [user.p_they()] isn't actually a janicart! That's a paddlin'!</span>")
-		if(user.mind.get_skill_level(/datum/skill/cleaning) >= SKILL_LEVEL_LEGENDARY) //Janny janny janny janny janny
+		if(user.mind.get_skill_level(/datum/skill/cleaning) >=SKILL_LEVEL_LEGENDARY) //Janny janny janny janny janny

```

Comment:
```suggestion
		if(user.mind.get_skill_level(/datum/skill/cleaning) >= SKILL_LEVEL_LEGENDARY) //Janny janny janny janny janny
```
Why would you make a change like this?

Reply:
Happened inadvertently when I was doing a ctrl+f find/replace, looks like i didn't catch 2 of these. Oops!